lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -377,6 +382,7 @@ class TripBasicsPane extends Component<TripBasicsProps, State> {
           <FormGroup validationState={errorStates.tripName}>
             <ControlLabel htmlFor="tripName">
               <FormattedMessage id="components.TripBasicsPane.tripNamePrompt" />
+              {!isReadOnly && <RequiredIndicator>{' *'}</RequiredIndicator>}
```

Comment:
Instead of using an additional space just for padding, please use a CSS margin of equivalent width for the corresponding `span` element. (2 instances)